js/client-battle.js

```diff
@@ -9,6 +9,8 @@
 		initialize: function (data) {
 			this.me = {};
 
+			this.isSideRoom = Tools.prefs('righthandbattles');
```

Comment:
`rightpanelbattles`, maybe? "hand" seems like a nonsequitur, even though I know what you're going for.